CHANGES.rst

```diff
@@ -1,16 +1,18 @@
 Changes
 =======
 
-Version 0.9.0 (UNRELEASED)
+Version 0.9.0 (2022-12-13)
 ---------------------------
 
 - Adds new ``launcher_url`` column to the ``Workflow`` table to store the remote origin of workflows submitted via Launch on REANA.
```

Comment:
... to store the remote origin of workflows submitted via the Launch-on-REANA functionality.